static/app/components/orgRole.tsx

```diff
@@ -0,0 +1,123 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import ExternalLink from 'sentry/components/links/externalLink';
+import Link from 'sentry/components/links/link';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconInfo} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {Member, Organization} from 'sentry/types';
+import {getEffectiveOrgRole} from 'sentry/utils/orgRole';
+
+export const OrgRoleInfo = ({
+  organization,
+  member,
+}: {
+  member: Member;
+  organization: Organization;
+}) => {
+  const {orgRoleList} = organization;
+  const {orgRole, orgRolesFromTeams} = member;
+
+  const orgRoleFromMember = orgRoleList.find(r => r.id === orgRole);
```

Comment:
Just saw this, I think everything is fine in here without a `useMemo`. `useMemo` is an optimization, so you can follow the usual advice here

> [Premature optimization is the root of all evil](https://wiki.c2.com/?PrematureOptimization)

Basically you really don't need a `useMemo` unless you **know for a fact** this will be slow. This is also a good article https://maxrozen.com/understanding-when-use-usememo#when-to-use-usememo

Basically any time you are rendering a relatively large N of something, you might want it